styles.css

```diff
@@ -50,10 +66,123 @@ h3 {
   text-transform: uppercase;
 }
 
-#social {
+li {
   list-style: none;
+}
+
+.social {
   display: flex;
   justify-content: space-between;
   margin-right: 170px;
   margin-left: -14px;
 }
+
+.project {
+  display: flex;
```

Comment:
@markotuya0 Kindly change from `display: flex` to `display: grid` as per project requirement.